guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>Can we have a newline before each line of the item type, to distinguish the item types (of the next category) and item names (of the current category)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>The display in ReviewNB does not preserve the formatting from the Notebook. I have the ouput grouped so there is a new line separating each Item Type group...and the individual items are listed with an indent. It is more legible in the notebook itself. I'll add a screenshot in GitHub. Let me know if that makes it clearer:</p><p> </p><p>Item Type</p><p>------------------</p><p>    Item1</p><p>    item2</p><p> </p><p>Item Type2</p><p>------------------ </p><p>    ItemType2_1</p><p>    ItemType2_2</p><p>    ItemType2_3</p><p> </p><p>user_inv</p>